src/compiler/crystal/tools/playground/views/_index.html

```diff
@@ -6,7 +6,7 @@
 
 <div class="row">
   <div class="col s7">
-    <small class="crystal-version grey-text text-lighten-1"></small>
+    <button id="runFormatterBtn" type="button" class="code-formatter-btn" title="Run code formatter">Run Formatter</button>
```

Comment:
Oh yes I like it there too. Although the heavy black is too contrasting from the show output. I'll make a change.